apps/obs-web/src/views/ErrorManagement.vue

```diff
@@ -113,72 +134,110 @@ import { get, set } from 'lodash';
 import _ from 'lodash';
 
 import { getTripsApiTrip, getCatchApiCatch } from '@boatnet/bn-common';
+import { authService } from '@boatnet/bn-auth/lib';
 
 export default createComponent({
     setup(props, context) {
+        const masterDB: Client<any> = couchService.masterDB;
         const columns: any = [
-            {field: 'name', header: 'Name' },
-            {field: 'description', header: 'Description'},
-            {field: 'program', header: 'Program'},
-            {field: 'isActive', header: 'Active?'},
-            {field: 'onEntry', header: 'On Entry?'},
-            {field: 'tripCheck', header: 'Trip Check?'},
+            {field: 'name', header: 'Name', isEditable: true },
+            {field: 'description', header: 'Description', isEditable: true },
+            {field: 'checkType', header: 'Type' },
+            {field: 'isActive', header: 'Active' },
```

Comment:
What does active mean? Does this mean it is active across all categories?

Reply:
yeah, the thought is, there's no way to delete an error - so marking it isActive: false would turn it off for all programs and validation types, but it could easily be turned back on.